test/cpp/end2end/client_lb_end2end_test.cc

```diff
@@ -3140,20 +3200,19 @@ TEST_F(WeightedRoundRobinTest, CallAndServerMetric) {
   response_generator.SetNextResolution(GetServersPorts(),
                                        kServiceConfigPerCall);
   // Send requests with per-call reported EPS/QPS set to 0/100.
-  // This should override per-server QPS above and give 1/4:1/2:1/3 = 3:6:4 WRR
-  // picks.
+  // This should give 1/2:1/3:1/15 = 15:10:2 WRR picks.
   EchoRequest request;
   // We cannot override with 0 with proto3, so setting it to almost 0.
   request.mutable_param()->mutable_backend_metrics()->set_eps(
       std::numeric_limits<double>::min());
   request.mutable_param()->mutable_backend_metrics()->set_rps_fractional(100);
   ExpectWeightedRoundRobinPicks(DEBUG_LOCATION, stub,
-                                /*expected_weights=*/{3, 6, 4},
+                                /*expected_weights=*/{15, 10, 2},
                                 /*total_passes=*/3, &request);
   // Now send requests without per-call reported QPS.
-  // This should change WRR picks back to 1:2:4.
+  // This should change WRR picks back to 2:1:8.
```

Comment:
This comment doesn't match the numbers below.

Reply:
Fixed.